### PR TITLE
HBW-81 Pass locale for translations from environment

### PR DESCRIPTION
**Pull request by sugarfree1**

### CODE REVIEW

**Comment by TimAle** [comment]
I'm not sure, but maybe it will better to make something like this in translations mixin:
```
getTranslatorForLocale (locale) {
  return (key, vars) => Translator.translate(key, locale, vars);
}
```
and get translator for locale from `this.props.env` once per component?

**Comment by sugarfree1** [comment]
Updated

**Comment by sugarfree1** [comment]
Updated

File: app/javascript/packs/hbw.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
-import 'babel-polyfill';
+/* eslint-disable global-require */
+
 import 'url-polyfill';
 import 'whatwg-fetch';
 import 'modulejs';
 import 'hbw';
+
+if (!global._babelPolyfill) {
```
Comment (TimAle) [inline app/javascript/packs/hbw.js]: I think you must load polyfill before all other stuff.

**Comment by sugarfree1** [comment]
Updated

**Comment by sugarfree1** [comment]
Updated

**Comment by sugarfree1** [comment]
Updated

**Pull request merged**